cupy/array_api/tests/test_set_functions.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from hypothesis import given
```

Comment:
As NumPy started to use hypothesis maybe we should consider using it too.
https://hypothesis.readthedocs.io/en/latest/index.html

Reply:
Feel free to ping me on any related issues/PRs if you need help with adopting Hypothesis.